gcn/notices/core/Statistics.schema.json

```diff
@@ -9,6 +9,14 @@
       "type": "number",
       "description": "False alarm rate: the rate of occurrence of non-astrophysical events that are of the same intensity or significance as the current event [Hz]"
     },
+    "net_count_rate": {
+      "type": "number",
+      "description": "Net count rate of the transient above the background [counts/s]"
+    },
+    "backgound_count_rate": {
+      "type": "number",
+      "description": "Count rate of the background for the transient [counts/s]"
```

Comment:
Same comment about rate_duration and rate_energy_range as in the net_count_rate.